### PR TITLE
nextSequenceToken that return by cloudwatch logs putLogEvents should be string, not int.

### PR DESCRIPTION
**Pull request by chrisLeeTW**
according to aws api docs (https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html#CWL-PutLogEvents-request-sequenceToken), nextSequenceToken's type should be string, not int.

### CODE REVIEW

File: moto/logs/models.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def put_log_events(self, log_group_name, log_stream_name, log_events, sequence_t
         self.events += [LogEvent(self.lastIngestionTime, log_event) for log_event in log_events]
         self.uploadSequenceToken += 1
 
-        return self.uploadSequenceToken
+        return "SEQ-{:010d}".format(self.uploadSequenceToken)
```
Comment (sfdb) [inline moto/logs/models.py]: Hey why did you add "SEQ-" here to the string? This is not what I'm getting back from the API, also in the doc page you linked it just shows as:

{
  "nextSequenceToken": "49536701251539826331025683274032969384950891766572122113"
}

Comment (chrisLeeTW) [reply moto/logs/models.py]: thanks for correction, fixed already.

**Comment by spulec** [comment]
Thanks!
